README.md

```diff
@@ -58,6 +58,25 @@ will use:
 ASDF_POSTGRES_VERSION=13.5 bundle install
 ```
 
+#### PostgreSQL on Linux
+
+With `asdf`, the default Unix socket is at `/tmp` and the default user is
+`postgres`. Rails expects the socket to be at `/run/postgresql` and the user to
+be your user (`whoami`). To fix these issues:
+
+```sh
+sudo mkdir -p /run/postgresql
+sudo chown $USER: /run/postgresql
+psql -U postgres
+> CREATE ROLE your-user-here LOGIN SUPERUSER;
+```
+
+When running PostgreSQL via `pg_ctl start`, specify the socket:
+
+```sh
+pg_ctl start -o "-k /run/postgresql"
+```
+
```

Comment:
Would it be worth modifying `bin/db` to do this? It'd probably require an OS check, etc.

Reply:
Let's not since I'm not sure everyone would choose to use `asdf` for Postgres anyway. :thinking: